spec/eof.md

```diff
@@ -233,12 +234,12 @@ The following instructions are introduced in EOF code:
         - loads the initcode EOF container from the transaction `initcodes` array which hashes to `tx_initcode_hash`
             - fails (returns 0 on the stack) if such initcode does not exist in the transaction, or if called from a transaction of `TransactionType` other than `INITCODE_TX_TYPE`
                 - caller's nonce is not updated and gas for initcode execution is not consumed. Only `TXCREATE` constant gas was consumed
-        - just before deducting hashing charge as in `EOFCREATE`, does following extra steps:
-            - deducts `2 * ((initcontainer_size + 31) // 32)` gas (EIP-3860 charge)
+        - in addition to hashing charge as in `EOFCREATE`, deducts `2 * ((initcontainer_size + 31) // 32)` gas (EIP-3860 charge)
+        - after checking call depth limit and whether caller balance is enough to transfer `value`:
```

Comment:
maybe safer to anchor this as "just before executing the container..."? The call depth step seems less "stable" to me, while the execution step is quite certain to remain at this spot.

Reply:
I applied this change.